.gitignore

```diff
@@ -1,4 +1,5 @@
 .gradle
+.idea
```

Comment:
You can also use [global gitignore](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) to ignore your personal files, regardless of repo, e.g. also including OS-specific and editor-specific stuff.